openarena.txt

```diff
@@ -4,7 +4,7 @@ Version = 0.8.8
 License = GPL
 Description = A community-produced deathmatch FPS game based on GPL idTech3 technology.
 Category = 4
-URLSite = https://www.openarena.ws/
+URLSite = https://web.archive.org/web/20250613102245if_/http://openarena.ws/smfnews.php
 URLDownload = https://download.tuxfamily.org/openarena/rel/088/openarena-0.8.8.zip
```

Comment:
I guess this link is also broken: https://download.tuxfamily.org/openarena/rel/088/openarena-0.8.8.zip